src/libPersistence/Retriever.cpp

```diff
@@ -42,7 +42,7 @@ void Retriever::RetrieveDSBlocks(bool& result)
     std::list<DSBlockSharedPtr> blocks;
     if (!BlockStorage::GetBlockStorage().GetAllDSBlocks(blocks))
     {
-        LOG_GENERAL(INFO, "RetrieveDSBlocks skipped or incompleted");
+        LOG_GENERAL(WARNING, "RetrieveDSBlocks skipped or incompleted");
         result = false;
```

Comment:
What is the purpose of result variable? You don't return the result in this case.

Reply:
It's a passed in reference, this function is passed into a new thread.

Reply:
ok